pkg/cvo/status.go

```diff
@@ -180,8 +181,16 @@ func (optr *Operator) syncStatus(ctx context.Context, original, config *configv1
 
 	now := metav1.Now()
 	version := versionString(status.Actual)
+	desired := configv1.Release{
+		Version: status.Actual.Version,
+		Image:   status.Actual.Image,
+	}
+	if availableUpdates != nil && desired.Image == availableUpdates.Current.Image {
+		desired.URL = availableUpdates.Current.URL
+		desired.Channels = availableUpdates.Current.Channels
```

Comment:
Are you just setting these for completeness, in case they are ever used in `mergeOperatorHistory` for example?

Reply:
Yup, because `mergeOperatorHistory` uses them today [for `status.desired`][1].

[1]: https://github.com/openshift/cluster-version-operator/blob/e49df43aaa578f0f420337bc28b7fc0f8cf35c09/pkg/cvo/status.go#L131